includes/api/class-wc-admin-rest-setting-options-controller.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * REST API Setting Options Controller
+ *
+ * Handles requests to /settings/{option}
+ *
+ * @package WooCommerce Admin/API
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Setting Options controller.
+ *
+ * @package WooCommerce Admin/API
+ * @extends WC_REST_Setting_Options_Controller
+ */
+class WC_Admin_REST_Setting_Options_Controller extends WC_REST_Setting_Options_Controller {
```

Comment:
Could we skip this controller override if we weren't turning an array into a comma delimited string?

Reply:
This is a tricky one.  The reason for this and the controller override is the way that WC Settings handles `multiselect` settings.  It expects an array of values and the only way to provide this is using the same query params with brackets to pass the value:

`value[]=option1&value[]=option2`

Passing the above string through `apiFetch` doesn't work since the brackets get escaped and unfortunately we also can't pass them via the data object since we can't provide the same value twice:

```js
const data = {
    value: 'option1',
    value: 'option2',
}
```

I'm open to other solutions and maybe I'm missing something obvious that handles this better.

Reply:
These overrides will eventually become the new core endpoints/core functionality -- so should we instead use this opportunity to change the handling of multiselect settings to look for `value0`, `value1` instead of comma handling?

Reply:
For some reason I couldn't "suggest a change" for this PR, so here's a branch showing everything working without juggling the array around as a string: https://github.com/woocommerce/wc-admin/pull/1389